en/directives/open-source-software.md

```diff
@@ -1,7 +1,5 @@
 # Directive on open source software
```

Comment:
We'll need to be consistent with the use of terms: open source software vs Open Source Software

Reply:
French has disappeared...